plugin/kubernetes/controller.go

```diff
@@ -410,3 +415,18 @@ func (dns *dnsControl) GetNamespaceByName(name string) (*api.Namespace, error) {
 	}
 	return v1ns, nil
 }
+
+func (dns *dnsControl) Modified() int64 {
```

Comment:
This doesn't actually tell you if it is modified? Should we just the `Serial(state request.Request) uint32` for this? Or change that interface?